api_docs/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 262**:
+
+* [`GET /users/{user_id}/status`](/api/get-user-status): Added
+  a new endpoint to get a specified user's currently set
+  [status](/help/status-and-availability).
```

Comment:
This is not an API change, just an API documentation change, and so does not belong here.

Reply:
Oh, it is, just the PR title is misleading :)